symfony/plugins/orangehrmTimePlugin/Dao/TimesheetDao.php

```diff
@@ -226,28 +226,18 @@ public function getTimesheetActionLogById($timesheetActionLogId)
     }
 
     /**
-     * Get Timesheet by given Start Date And Employee Id
-     * @param $starDate , $employeeId
-     * @return Timesheet
+     * @param DateTime $startDate
+     * @param int $employeeId
+     * @return Timesheet[]
      */
-    public function getTimesheetByStartDateAndEmployeeId($startDate, $employeeId)
+    public function getTimesheetByStartDateAndEmployeeId(DateTime $startDate, int $employeeId): array
```

Comment:
```suggestion
    public function getTimesheetByStartDateAndEmployeeId(DateTime $startDate, int $empNumber): array
```

Reply:
Method removed because we have used from detailedTimesheet object in service